install.sh

```diff
@@ -69,7 +69,7 @@ install_nvm_from_git() {
     mkdir -p "$NVM_DIR"
     command git clone "$(nvm_source git)" "$NVM_DIR"
   fi
-  cd "$NVM_DIR" && command git checkout --quiet $(nvm_latest_version)
+  cd "$NVM_DIR" && command git checkout --quiet "$(nvm_latest_version)"
```

Comment:
fwiw, `nvm_latest_version` can't possibly return anything with spaces, so the quoting here isn't necessary, but it can't hurt